node/src/GlideClusterClient.ts

```diff
@@ -785,6 +785,7 @@ export class GlideClusterClient extends BaseClient {
      *
      * The command will be routed to all primary nodes, unless `route` is provided.
      *
+     * When `route` is set to `allNodes`, the `INFO` command may return partial data, as the response from each node might be incomplete due to factors like network conditions or node load. This can lead to varying results across different executions.
```

Comment:
Why this happen? Does this happen when command is routed to all primaries? Does this happen with other commands? Does this happen with java client?

Reply:
this is not new, this is happen due to the Special policy ```ResponsePolicy::Special```